src/app/model.ts

```diff
@@ -7,16 +7,16 @@ export interface BaseEntity {
 }
 
 export interface ListResult extends BaseEntity {
-    totalCount: number;
     items: Array < {} >
     ;
+    totalCount: number;
```

Comment:
I would really love it if we could break up this huge file into multiple, smaller files.